gdcdictionary/schemas/analyte.yaml

```diff
@@ -50,37 +49,37 @@ uniqueKeys:
 
 properties:
 
-  $ref:
-    - "_definitions.yaml#/ubiquitous_properties"
+  $ref: "_definitions.yaml#/ubiquitous_properties"
 
   a260_a280_ratio:
-    $ref:
-      - "_terms.yaml#/a260_a280_ratio/common"
+    term:
+      $ref: "_terms.yaml#/a260_a280_ratio/common"
     type: number
     minimum: 0
 
   amount:
-    $ref:
-      - "_terms.yaml#/amount/common"
+    term:
+      $ref: "_terms.yaml#/amount/common"
     type: number
     minimum: 0
 
   analyte_quantity:
-    $ref:
-      - "_terms.yaml#/analyte_quantity/common"
+    term:
+      $ref: "_terms.yaml#/analyte_quantity/common"
     type: number
     minimum: 0
 
   analyte_type:
-    $ref:
-      - "_terms.yaml#/analyte_type/common"
+    term:
+      $ref: "_terms.yaml#/analyte_type/common"
     enum:
       - cfDNA
       - DNA
       - EBV Immortalized Normal
       - FFPE DNA
       - FFPE RNA
       - GenomePlex (Rubicon) Amplified DNA
+      - m6A enriched RNA
```

Comment:
@tlicht3 @ZhenyuZ  I'm thinking we can change this to a capital E to match the other capitalizations. Sound good?